config.py

```diff
@@ -0,0 +1,25 @@
+import os
```

Comment:
would it be better if you named this file something more than just "config" because config itself just seems like a general config filename for everything.

Reply:
This file is `config.py` by convention for the flask app